src/rqt_plot/plot_widget.py

```diff
@@ -114,10 +114,9 @@ def get_plot_fields(node, topic_name):
         no_field_error_msg = base_error_msg + f"'{name}' is not a field of '{topic_type_str}'"
 
         try:
-            slot_index = current_message_class.__slots__.index(f'_{name}')
+            current_type = current_message_class.get_fields_and_field_types()[name]
```

Comment:
Actually, sorry.  I just realized that this is wrong.  This is returning the type from `_fields_and_field_types`, but we really need the `rosidl_parser.definition` type from `SLOT_TYPES`.

Reply:
Oh you are right, nice catch.